pavelib/assets.py

```diff
@@ -636,6 +636,30 @@ def collect_assets(systems, settings):
         print("\t\tFinished collecting {} assets.".format(sys))
 
 
+def execute_compile_sass(args):
+    """
+    Construct django management command compile_sass (defined in theming app) and execute it.
+    Args:
+        args: command line argument passed via update_assets command
+    """
+    for sys in args.system:
+        options = ""
+        options += " --themes " + " ".join(args.themes) if args.themes else ""
+        options += " --themes-dir=" + args.themes_dir if args.themes_dir else ""
+        options += " --debug" if args.debug else ""
+
+        sh(
```

Comment:
Thanks for the explanation @saleem-latif. Maybe add some comments to this effect so that future developers won't be as confused as I was!